plugins/inputs/docker/docker.go

```diff
@@ -435,6 +435,22 @@ func (d *Docker) gatherContainer(
 			}
 		}
 	}
+	if info.State != nil {
+		statefields := map[string]interface{}{
+			"status":     info.State.Status,
+			"running":    info.State.Running,
+			"paused":     info.State.Paused,
+			"restarting": info.State.Restarting,
```

Comment:
I don't think we need to save the running, paused, restarting, dead values, since this is all encompassed by the status field.

Reply:
Fair point . I will remove them....